2017/09.md

```diff
@@ -0,0 +1,341 @@
+
+---
+
+CMS: Joomla
+
+* Version 3.8 released September 19th.
+* LDAP injection vulnerability found in v3.7.5
+
+[https://blog.ripstech.com/2017/joomla-takeover-in-20-seconds-with-ldap-injection-cve-2017-14596/](https://blog.ripstech.com/2017/joomla-takeover-in-20-seconds-with-ldap-injection-cve-2017-14596/)
+
+---
+
+![fit](https://blog.ripstech.com/videos/joomla-poc.mp4)
+
+---
+
+# IDE: PHPStorm
+
+2017.2.3
+
+* UML diagrams for classes
+* Support for Docker Compose 3.0 and 3.1 files (3.2, 3.3 soon)
+
+2017.3 EAP
+
+* Support of nullability in refactoring (in tandem with PHP 7.1)
+* Editor-based REST client
+
+`https://blog.jetbrains.com/phpstorm/`
+
+---
+
+# IDE: Sublime Text
+
+3.0 Released!
+
+* Refreshed UI theme (for those of you who don't use Material Theme)
+* New color schemes, and a ...new icon (not happy about it)
+* Significantly faster than ST2
+* ST2 licenses carry over to ST3 if purchased after 2013
+
+`https://code.visualstudio.com/updates`
+
+---
+
+# PHP Conferences - August
+
+**Laracon EU**
+28-30, Amsterdam
+https://laracon.eu/2017/
+
+* FREE! All talks available FREE on youtube - [https://laravel-news.com/laracon-eu-2017-videos](https://laravel-news.com/laracon-eu-2017-videos)
+
+---
+
+# PHP Conferences - September
+**Pacific Northwest PHP Conference**
+September 7–9, Seattle, Washington
+pnwphp.com
+
+**Madison PHP Conference 2017**
+September 22–23, Madison, WI
+2017.madisonphpconference.com
+
+---
+
+# Nomad PHP
+
+** FREE MEETING **
+
+Profiling PHP7 Applications
+By Paris Liakos
+
+**US**
+Zero to Hero: API Development
+By Matt Trask
+
+**Europe**
+Iterators and Generators
+By Eli White
+
+`https://nomadphp.com`
+
+---
+
+# Upcoming regional events
+
+Oct 7-8 - NCDevCon - http://ncdevcon.com
+
+Oct 23-24 - All Things Open - http://allthingsopen.org
+
+Nov 15-16 - PHP World - http://world.phparch.com (I have a discount code!)
+
+---
+
+# Notable News & Articles
+
+* https://murze.be/2017/08/laravel-5-5-means-packages/
+* https://www.amitmerchant.com/Laravel-Eager-Loading-Load-Vs-With/
+* https://mattstauffer.co/blog/laravel-collections-higher-order-messaging-and-when-method-in-laravel-5-4
+* https://framework.zend.com/blog/2017-08-17-php72-argon2-hash-password.html
+* https://www.sitepoint.com/twig-popular-stand-alone-php-template-engine/
+* http://www.omniceps.com/latte-best-php-templating-engine/
+* https://medium.com/@pawel.mikolajczuk/custom-caching-for-twig-f9a7303cebdd
+
+---
+
+# Next Month @ TrianglePHP
+
+**TITLE**
+Tombstones, Vampires, and Discovering Dead Code by Chris Gmyr
+
+Thursday, **October 19th**
+6pm
+Atlantic BT
+
+[https://www.meetup.com/trianglephp/events/238029321/](https://www.meetup.com/trianglephp/events/238029321/)
+
+---
+
+# Have a Job/Need a Job
+
+**Who's hiring?**
+**Who's looking?**
+**What are you looking for?**
+
+---
+
+>Thank you to Atlantic BT for hosting and refreshments!
+
+---
+
+# Tonight @ TrianglePHP
```

Comment:
This schedule is going to be a little different - the presentation is actually going to be before the "What's New" Do you want to move this to the first slide?

Reply:
If the latest in php prez is going last, I can take out this part entirely